app/src/util/tools/formatting.ts

```diff
@@ -33,8 +33,13 @@ export const formatNumber = (number: string, decimals = 2): string => {
   if (number.length < 1) {
     return `0${decimals > 0 ? '.' + '0'.repeat(decimals) : ''}`
   }
-  if (Number(number) < 0.01 && Number(number) > 0) {
-    return '<0.01'
+
+  if (Number(number) > 0) {
+    for (let i = 1; i < 10; i++) {
+      if (Number(number) < Number('0.' + '0'.repeat(i) + '1') && decimals === i + 1) {
+        return '<0.' + '0'.repeat(i) + '1'
+      }
+    }
```

Comment:
@Frankaus Why are we making this loop? I think because we have decimals we can just determine which evaluation we need to do and use the value for evaluation to be based on number of decimals....